setup.py

```diff
@@ -58,6 +58,7 @@
             'ipywidgets',
             'sphinx-issues',
             'sphinx-copybutton>=0.3.2',
+            'sphinx-togglebutton>=0.3.0',
```

Comment:
Setting to v0.3.0 as lower bound as that's what is available on PyPI today: https://pypi.org/project/sphinx-togglebutton/0.3.0/